definitions/npm/base64-arraybuffer_v1.x.x/flow_v0.83.x-/test_base64-arraybuffer_v1.x.x.js

```diff
@@ -0,0 +1,20 @@
+import { describe, it } from 'flow-typed-test';
```

Comment:
I'm pretty sure you need to add this otherwise it doesn't typecheck
```suggestion
// @flow
import { describe, it } from 'flow-typed-test';
```